.env.test

```
@@ -0,0 +1,16 @@
+# Sets the location of the flask application object: https://flask.palletsprojects.com/en/2.3.x/cli/#application-discovery
+# Needed because of the presence of the "todo_app" folder
+FLASK_APP=todo_app/app
+
+# Turn on debug mode (which enables reloading on code changes and the interactive debugger: https://flask.palletsprojects.com/en/2.3.x/config/#DEBUG)
+FLASK_DEBUG=true
+
+# Required to use Flask's session feature (for storing todos in a browser cookie in the first exercise): https://flask.palletsprojects.com/en/2.3.x/config/#SECRET_KEY
+SECRET_KEY=secret-key
+
+# Add you Trello API credentials here:
+TRELLO_API_KEY=<insert test key here>
+TRELLO_API_TOKEN=<insert test token here>
+TRELLO_BOARD_ID=<insert test Board ID here>
+TRELLO_TODO_LIST_ID=<insert test List ID here>
+TRELLO_DONE_LIST_ID=<insert test List ID here>
```

Since these are test variables, actually we don't want the user to add anything in here - so this works even without changing it, but probably nicer is to make it clear these are just set to any old test variable:
```suggestion
TRELLO_API_KEY=fake-api-key
TRELLO_API_TOKEN=fake-api-token
TRELLO_BOARD_ID=fake-board-id
TRELLO_TODO_LIST_ID=fake-todo-list-id
TRELLO_DONE_LIST_ID=fake-done-list-id
```